bokeh/models/plots.py

```diff
@@ -240,6 +240,23 @@ def add_tile(self, tile_source, **kw):
         self.renderers.append(tile_renderer)
         return tile_renderer
 
+    def add_dynamic_image(self, image_source, **kw):
```

Comment:
I think this unnecessarily bloats `Plot`'s API (the same goes for `add_tile()` that was added earlier).